tests/test_models.py

```diff
@@ -456,6 +456,31 @@ def test_invalid_expressions_access(self, f_obj):
 
 
 class TestExpressions:
+    def test_bulk_update(self):
+        if VERSION[:2] < (2, 2):
+            pytest.skip("skipping tests for Django < 2.2")
+
+        assert ModelWithVanillaMoneyField.objects.filter(integer=0).count() == 0
+        assert ModelWithVanillaMoneyField.objects.filter(integer=1).count() == 0
+        ModelWithVanillaMoneyField.objects.create(money=Money(1, "USD"), integer=0)
+        ModelWithVanillaMoneyField.objects.create(money=Money(2, "USD"), integer=1)
+        first_instance = ModelWithVanillaMoneyField.objects.get(integer=0)
+        second_instance = ModelWithVanillaMoneyField.objects.get(integer=1)
+        assert first_instance.money == Money(1, "USD")
+        assert second_instance.money == Money(2, "USD")
+        first_instance.money = Money(3, "RUB")
+        second_instance.money = Money(4, "UAH")
+        second_instance.second_money = Money(5, "BYN")
+        ModelWithVanillaMoneyField.objects.bulk_update([first_instance, second_instance], ("money", "second_money"))
+        ModelWithVanillaMoneyField.objects.get(integer=0).money == Money(3, "RUB")
+        ModelWithVanillaMoneyField.objects.get(integer=1).money == Money(4, "UAH")
+        ModelWithVanillaMoneyField.objects.get(integer=1).second_money == Money(5, "BYN")
+
+    def test_date_lookup(self):
```

Comment:
This one is identical to the test on line 180 in this file - could you, please, remove it?